tutorials/focus-mode/focus-mode.css

```
@@ -0,0 +1,12 @@
+/* 
+Copyright 2021 Google LLC
+Use of this source code is governed by a BSD-style
+license that can be found in the LICENSE file or at
+https://developers.google.com/open-source/licenses/bsd 
+*/
+
+body > .scaffold > :is(top-nav, navigation-rail, side-nav, footer),
+main > :not(:last-child),
+main > :last-child > navigation-tree {
```

Might want to also hide the right navigation menu when viewing the docs in a wide window.

```suggestion
main > :last-child > navigation-tree,
main .toc-container {
```